src/containers/Debug/.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
@bensjx hi, what does this launch.json file do?

Reply:
Hi, this file is not needed, i will be deleting it